docs/cli/index.rst

```diff
@@ -21,3 +21,198 @@ To execute the command line tool type ``ntc_rosetta`` after installing the libra
      print-parser  Prints a tree representation of a parser/translator.
 
 The command line tool is self-documented and you can check it's documentation using the ``--help`` flag.
+
+Lint
+####
+ntc_rosetta lint will lint the parsers and translators and offers several options to be specified::
+
+   $ ntc_rosetta git:(docs) ✗ ntc_rosetta lint --help
+   Usage: ntc_rosetta lint [OPTIONS] [FILEPATHS]...
+
+     Lint files/folders with Parsers/Translators:
+
+     Errors/Warning message codes:
+
+       - E001: path is empty
+
+       - E101: schema path couldn't be found
+
+       - E102: schema path is invalid
+
+       - W001: children is missing
+
+       - W101: class attribute doesn't belong to the model
+
+   Options:
+     -i, --ignore TEXT             ignore error/warning codes - e.g. specify multiple -i to ignore more than one code
+
+     -m, --model [openconfig|ntc]  model to lint - default: openconfig
+
+     -j, --json / -t, --text       output format - default: text
+
+     --help                        Show this message and exit.
+
+It will define the FILEPATHS as the current working directory. If we're in the root of **ntc-rosetta** then we will need to either move to the **ntc_rosetta** folder or specify **ntc_rosetta** when executing the command::
```

Comment:
I trust folks will figure this out, but maybe for a little more clarity:

> If we're in the repository root of **ntc-rosetta** (note the dash), then we will need ...

Reply:
Ok I updated and added an example of both so hopefully it helps clarify it. Don't want the users to make any assumptions so good catch.